mission1/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta property="og:title" content="판다 마켓" />
+    <meta property="og:description" content="일상의 모든 물건 을 거래해보세요" />
+    <meta property="og:url" content="/" />

```

Comment:
👍 칭찬
아직 배우지 않으신 메타 태그 추가해주신 것도 좋아요~ 다만 내용만 좀 고쳐주시면 더 좋을 것 같아요!

```suggestion
    <meta property="og:title" content="판다마켓" />
    <meta property="og:description" content="일상의 모든 물건을 거래해보세요" />
    <meta property="og:url" content="https://hongyohan.netlify.app/mission1/1" />
```